### PR TITLE
Update to latest runtime schema to support RequestId header optional

### PR DESCRIPTION
**Pull request by relango**
- Update to latest runtime schema to support RequestId header optional
- GDPR complaince changes to not log request payload and mask authentication tokens 
- Fixed security vulnerabilities in 3PP libraries
- Support for case insensitive Bot Response Http headers

### CODE REVIEW

File: src/main/java/com/salesforce/einsteinbot/sdk/client/BasicChatbotClientImpl.java [hunk 1]
```diff
@@ -232,9 +232,8 @@ private WebClient createWebClient(WebClient.Builder webClientBuilder) {
 
   private Consumer<ClientCodecConfigurer> createCodecsConfiguration(ObjectMapper mapper) {
     return clientDefaultCodecsConfigurer -> {
-      //isEnabled for LoggingJsonEncoder should be false if we don't want to Request Body for GDPR compliance.  TODO: Make this configurable in future
       clientDefaultCodecsConfigurer.defaultCodecs()
-          .jackson2JsonEncoder(new LoggingJsonEncoder(mapper, MediaType.APPLICATION_JSON, true));
+          .jackson2JsonEncoder(new LoggingJsonEncoder(mapper, MediaType.APPLICATION_JSON, false));
```
Comment (atulkc) [inline src/main/java/com/salesforce/einsteinbot/sdk/client/BasicChatbotClientImpl.java:236]: 👍

File: src/main/java/com/salesforce/einsteinbot/sdk/client/model/BotHttpHeaders.java [hunk 2]
```diff
@@ -36,18 +40,20 @@ public class BotHttpHeaders {
 
   private BotHttpHeaders(Multimap<String, String> headerValues) {
     Objects.requireNonNull(headerValues);
-    this.headerValues = headerValues;
+    headerValues
+        .entries()
+        .stream()
+        .forEach(e -> this.headerValues.put(e.getKey().toUpperCase(), e.getValue()) );
```
Comment (atulkc) [inline src/main/java/com/salesforce/einsteinbot/sdk/client/model/BotHttpHeaders.java:46]: Nit:
Usually proxies would convert header names to be all lower case. Would be good if we maintain it as all lower case.

**Review by atulkc — APPROVED** [review]
One minor comment but looks good otherwise.

File: src/main/java/com/salesforce/einsteinbot/sdk/client/model/BotHttpHeaders.java (hunk 2 above)
Comment (relango) [reply src/main/java/com/salesforce/einsteinbot/sdk/client/model/BotHttpHeaders.java]: Sure, updated to use lower case instead.

**Pull request merged**